coldfront/templates/email/project_join_request_approved.txt

```diff
@@ -1,31 +1,56 @@
-Dear {{ user.first_name }} {{ user.last_name }},
-
+Dear {{ user.first_name }} {{ user.last_name }},                                                                                                                                                                    
+ 
 Your request to join Project {{ project_name }} has been approved by its managers. A request to get access to the project on the cluster has been generated, and will be processed by admins.
-
-As you wait for your account to be processed and to get access to Savio, we strongly recommend, as we do with all new Savio users, that you see and familiarize yourself with our extensive Savio documentation here:
-
+ 
+As you wait for your account to be processed and to get access to Savio,, we strongly recommend that you see and familiarize yourself with our extensive Savio documentation here:
```

Comment:
There are two commas here (admittedly from the original text, but worth correcting).